Map runtime CoreJS 3 polyfills to native globals (#5475)
woocommerce/woocommerce-gateway-stripe: changelog.txt

* Dev - Add Jest tests for ECE payment flow
* Fix - Respect the iDEAL/Wero and Bancontact save toggle when Optimized Checkout is enabled
* Update - Improve express checkout load times by preloading the resources

> * Tweak - Reduce JS bundle sizes by externalizing core-js polyfills for features natively supported in modern browsers.

Reviewer: I have two comments here:
* We should remove the trailing `.` from the entry.
* I think we can simplify the description, and it may also be worth describing this as an Update due to the relevant impact.
```suggestion
* Tweak - Reduce JS bundle sizes by using native browser features instead of polyfills
```
